rest/context.go

```diff
@@ -325,6 +329,30 @@ func (ctx *gorillaRequestContext) NextURL() (string, error) {
 	return u.String(), nil
 }
 
+func (ctx *gorillaRequestContext) BuildURL(routeName, resourceID string) (string, error) {
+	r, ok := ctx.Request()
+	if !ok {
+		return "", fmt.Errorf("Unable to build next url: no request")
```

Comment:
I copied that portion directly from [NextURL()](https://github.com/Workiva/go-rest/blob/master/rest/context.go#L305-308) without much thought  :-)

Will update, though the error message should be clear that the problem is the lack of a request on the context, not the arguments.